CMakeLists.txt

```diff
@@ -41,23 +41,20 @@ macro(build_uncrustify)
   # Pinning to tip of master at the time of a desired bugfix
   set(uncrustify_version 45b836cff040594994d364ad6fd3f04adc890a26)
   # Get uncrustify 0.68.1
-  ExternalProject_Add(uncrustify-0.68.1
+  ExternalProject_Add(uncrustify-${uncrustify_version}
     GIT_REPOSITORY https://github.com/uncrustify/uncrustify.git
     GIT_TAG ${uncrustify_version}
     GIT_CONFIG advice.detachedHead=false
+    # Suppress git update due to https://gitlab.kitware.com/cmake/cmake/-/issues/16419
+    # See https://github.com/ament/uncrustify_vendor/pull/22 for details
+    UPDATE_COMMAND ""
```

Comment:
As we just found out, https://github.com/eProsima/foonathan_memory_vendor/blob/0e6174a7f28c45faf711030697a6c32547336acd/CMakeLists.txt#L79 also has this workaround.  So :crossed_fingers: this is the right solution.